src/index.ts

```diff
@@ -55,6 +55,13 @@ async function main() {
     workflow_ids.push(String(current_run.workflow_id));
   }
   console.log(`Found workflow_id: ${JSON.stringify(workflow_ids)}`);
+  const trigger_repo_id = (function () {
+    if (payload.workflow_run) {
+      return payload.workflow_run.head_repository.id;
+    } else {
+      return current_run.head_repository.id;
+    }
+  })();
```

Comment:
```suggestion
  const trigger_repo_id = payload.workflow_run
    ? payload.workflow_run.head_repository.id
    : current_run.head_repository.id;
```

You can use a ternary operator here